### DISCUSSION TITLE
skynode problem using different ways to create skynode

### DISCUSSION THREAD

**Discussion by cxhaizxm** [pelicanmapping/osgearth#1792] [Support Q&A]
Hi，today i encounter one strange problem. I use the code to create skyNode and let it attach the viewer, i can see double image. i think it is z-flighting, so i open the LogarithmicDepthBuffer, however it is not effect.
 But when i use the console to load sky,everything is oky. 
i use the osgearth 3.2 osg3.6.5 to test in win10 2017 64.The code and the strange phenomenon photo is below. 
I am hoping for your help.Thank you very much.

![image](https://user-images.githubusercontent.com/7930275/133188281-a304cf35-8279-4480-a95e-bc8ce74ba0f9.png)

![image](https://user-images.githubusercontent.com/7930275/133187869-ef9c23e9-fa18-440d-a758-3f94be8730b4.png)

**Comment by gwaldron** [comment]
What does your earth file look like? Can you provide a viewpoint where we might be able to reproduce the artifact?
